addons/sms/models/sms_template.py

```diff
@@ -86,6 +88,32 @@ def copy(self, default=None):
                        name=_("%s (copy)") % self.name)
         return super(SMSTemplate, self).copy(default=default)
 
+    def create_action(self):
```

Comment:
Naming: I would name it like other action, for example action_create_act_window, and action_remove_act_window for the other one ...